src/EditorFeatures/CSharpTest/Recommendations/ThisKeywordRecommenderTests.cs

```diff
@@ -526,5 +526,109 @@ void M()
     {
         var c = new C { x = 2, y = 3, $$");
         }
+
+        [WorkItem(725)]
+        [WorkItem(1107414)]
+        [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
+        public void InExpressionBodiedMembersProperty()
+        {
+            VerifyKeyword(@"
+class C
+{
+    int x;
+    int M => $$");
+        }
+
+        [WorkItem(725)]
+        [WorkItem(1107414)]
+        [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
+        public void InExpressionBodiedMembersProperty_1()
+        {
+            VerifyKeyword(@"
+class C
+{
+    int x;
+    int M => $$
```

Comment:
Take look at [VerifyKeyword](http://source.roslyn.io/#Roslyn.Services.Editor.CSharp.UnitTests/Recommendations/RecommenderTests.cs,1e641d3761d586fb|VerifyKeyword). Given a file with the caret indicated, it will run the test 
a) with exactly the specified text and caret position
b) with the specified text, with a /* */ comment inserted before the caret position
c) with the specified text only up to the caret position
d) the above, with the keyword partially typed.

The test `InExpressionBodiedMembersProperty` duplicates the (c) part of `InExpressionBodiedMembersProperty_1`, so you can get rid of it.